Stratis.Bitcoin.Features.LightWallet/LightWalletSyncManager.cs

```diff
@@ -220,10 +264,15 @@ public void SyncFromDate(DateTime date)
         /// <inheritdoc />
         public void SyncFromHeight(int height)
         {
+            if (height < 0)
+            {
+                throw new WalletException("Invalid block height");
```

Comment:
Can we have some logging here to tell us what happened?
Also please add the height in the exception.